beacon-chain/verification/data_column.go

```diff
@@ -257,17 +257,25 @@ func (dv *RODataColumnsVerifier) ValidProposerSignature(ctx context.Context) (er
 			continue
 		}
 
-		columnVerificationProposerSignatureCache.WithLabelValues("miss").Inc()
+		// Ensure the expensive signature verification is only performed once for
+		// concurrent requests for the same signature data.
+		if _, err, _ = dv.sg.Do(signatureData.string(), func() (any, error) {
```

Comment:
Could we avoid the new dependency by using [sync.OnceValues](https://pkg.go.dev/sync#OnceValues)?

Reply:
nm, I misunderstood what this type does (and mistook it as a new dep).